proto/gateway/service.proto

```diff
@@ -129,7 +130,8 @@ message GetFeatureResponse {
 message UpdateFeatureRequest {
   string comment = 1;
   string id = 2;
-  feature.Feature feature = 3;
+  google.protobuf.StringValue name = 3;
+  google.protobuf.StringValue description = 4;
```

Comment:
I changed how to pass values to be updated.
If we pass `feature.Feature` (previous commit), some fields can be updated (e.g., name, description, rules, etc.), and some fields cannot be updated (e.g., version, variation_type, etc.), which confuses users.
I set nullable fields to request so that users can choose what they want to update, and it is easy for backend to know what to update.
I start with the name and description. I'm adding other fields in the coming PRs.